ext/FileIOExt.jl

```diff
@@ -0,0 +1,34 @@
+module FileIOExt
+
+using BED, FileIO
+
+function BED.fileio_load(filepath::File{format"BED"})
+    
+    reader = open(BED.Reader, FileIO.filename(filepath))
+    records = try 
+        records = [record for record in reader]
```

Comment:
Does it work if round brackets are used here instead of square brackets? Round brackets will give you a generator instead of loading all of the records into an array.